auth-web/src/components/auth/BCSCPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
```

Comment:
I suggest camel-case for the filename (ie, BcscPanel.vue).

Reply:
ah definitely. Will do

Reply:
Haha, I meant everywhere it is used. But not worth a lot of time on this.

Reply:
Oops, sorry. Missed a few. Fix coming in next commit.